hw3/main.tf

```diff
@@ -51,11 +51,13 @@ resource "aws_iam_policy" "s3_policy" {
     {
       "Effect": "Allow",
       "Action": [
-        "s3:PutObject"
+        "s3:PutObject",
+        "dynamodb:PutItem"
       ],
       "Resource": [
         "arn:aws:s3:::my-first-bucket-ihor2",
-        "arn:aws:s3:::my-first-bucket-ihor2/*"
+        "arn:aws:s3:::my-first-bucket-ihor2/*",
+        "arn:aws:dynamodb:us-west-2:646632584624:table/my-table"
```

Comment:
It's better to refer to arn by `aws_dynamodb_table.my_table.arn`. You can use a lot of attributes from terraform resources; the full list you can find on corresponding page in "Attributes Reference" section, for example, https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/dynamodb_table#attributes-reference